mysql-public-db.tf

```diff
@@ -0,0 +1,91 @@
+# Reference to the PostgreSQL/MySql dedicated network external resources
+# ALREADY LOADED IN postgres-public-db.tf
+# data "azurerm_virtual_network" "public_db"
+
+# Dedicated subnet is reserved as "delegated" for the mysql server (same as the postgres) on the public network
+# not specificaly needed as per the mysql flexible server documentation :
+# Ref. https://learn.microsoft.com/en-us/azure/mysql/flexible-server/concepts-networking
+# but seem adequate to use Private access (VNet integration) https://learn.microsoft.com/en-us/azure/mysql/flexible-server/concepts-networking#choose-a-networking-option
```

Comment:
I followed the postgres architecture but it seem not mandatory for mysql, WDYT ?